modules/measure/src/test/java/com/opengamma/strata/measure/swaption/SwaptionTradeCalculationsTest.java

```diff
@@ -64,13 +68,30 @@ public void test_pv01() {
     CurrencyParameterSensitivities expectedPv01CalBucketed = pvParamSens.multipliedBy(1e-4);
     PointSensitivities pvVegaPointSens = pricer.presentValueSensitivityModelParamsVolatility(RTRADE, provider, VOLS);
     CurrencyParameterSensitivities expectedPv01VegaCal = VOLS.parameterSensitivity(pvVegaPointSens);
+    CurrencyParameterSensitivities expectedPv01VegaCalAdjusted = expectedPv01VegaCal.getSensitivities().stream()
```

Comment:
Is there no way to use `CurrencyParameterSensitivities.toBuilder().filterSensitivity(...) ` followed by `CurrencyParameterSensitivities.mapSensitivities(...)` ? If not, then maybe Strata needs an enhancement

Reply:
I've added a `mapSensitivities(BiFunction<ParameterMetadata, Double, Double>)` method to the builder.